README.md

```diff
@@ -114,6 +114,9 @@ to verify contracts, the enviornment variable should contain `ETHERSCAN_API_KEY`
 constructor arguments can be passed as follows:
```

Comment:
Because we use the word "arguments" here as a plural but for an example where there id only one argument, we might want to change the description.